### DISCUSSION TITLE
Solar radiation and photovoltaic system performance

### DISCUSSION THREAD

**Discussion by juliana-steinbach** [Depart-de-Sentier/sentier.dev#34] [Data suggestion]
### Data Source name

Photovoltaic Geographical Information System (PVGIS)

### Data Source Homepage URL

https://joint-research-centre.ec.europa.eu/photovoltaic-geographical-information-system-pvgis_en

### Data License

-

### Justify this Data Source Choice

Free and open access to photovoltaic (PV) electricity generation potential for different technologies and configurations.
Available in English, French, Italian, Spanish and German.
No registration
Extensive supporting documentation - see the links at the bottom of this page.
First time user? Check out the [Getting started](https://joint-research-centre.ec.europa.eu/want-make-best-solar-energy-where-you-live/getting-started-pvgis_en) section
APIs for fast, automated access needs.
Maps of solar resource and PV potential, by country or region, in ready to print files.

### Area of Application

Renewable electricity - Photovoltaic, PV, Solar

**Comment by mfastudillo** [comment]
I've also used the API of https://www.renewables.ninja/ in the past. Although the NC license is a bit limiting

**Reply by juliana-steinbach** [reply]
Thanks for sharing it!
